src/PrestaShopBundle/Twig/HookExtension.php

```diff
@@ -151,19 +152,18 @@ public function renderHooksArray($hookName, $hookParameters = array())
      *
      * @throws \Exception if the hookName is missing
      *
-     * @return string all listener's reponses, concatened in a simple string, ordered by the listeners' priorities
+     * @return string all listener's responses, concatenated in a simple string, ordered by the listeners' priorities
      */
     public function renderHook($hookName, $hookParameters = array())
     {
         if ($hookName == '') {
             throw new \Exception('Hook name missing');
         }
-        $hookRenders = $this->hookDispatcher
+
+        return $this->hookDispatcher
             ->dispatchRenderingWithParameters($hookName, $hookParameters)
-            ->getContent()
+            ->outputContent()
         ;
-
-        return empty($hookRenders) ? '' : implode('<br class="hook-separator" />', $hookRenders);
```

Comment:
time to remove this ugly hack: add an HTML separator between 2 modules displaying an hook?

I mean: seriously? o_O